SPECS/intel-ipsec-mb/intel-ipsec-mb.spec

```
@@ -0,0 +1,150 @@
+#
+#     * Redistributions of source code must retain the above copyright notice,
+#       this list of conditions and the following disclaimer.
+#     * Redistributions in binary form must reproduce the above copyright
+#       notice, this list of conditions and the following disclaimer in the
+#       documentation and/or other materials provided with the distribution.
+#     * Neither the name of Intel Corporation nor the names of its contributors
+#       may be used to endorse or promote products derived from this software
+#       without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# Versions numbers
+BuildRequires:  gcc >= 4.8.3
+BuildRequires:  make
+BuildRequires:  nasm >= 2.14
+ExclusiveArch:  x86_64
+
+%description
+IPSEC cryptography library optimized for Intel Architecture
+
+%package -n intel-ipsec-mb-devel
+Summary:        IPSEC cryptography library optimized for Intel Architecture
+Group:          Development/Tools
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+ExclusiveArch:  x86_64
+
+%description -n intel-ipsec-mb-devel
+IPSEC cryptography library optimized for Intel Architecture
+
+For additional information please refer to:
+https://github.com/intel/%{githubname}
+
+%prep
+%autosetup -n %{githubfull}
+
+%if 0%{?rhel} && 0%{?rhel} < 8
+%ldconfig_post
+
+%ldconfig_postun
+%endif
+
+%build
+cd lib
+make EXTRA_CFLAGS='%{optflags}' %{?_smp_mflags}
+
+%install
+
+# Install the library
+install -d %{buildroot}/%{_includedir}
+install -m 0644 %{_builddir}/%{githubfull}/lib/intel-ipsec-mb.h %{buildroot}/%{_includedir}
+install -d %{buildroot}/%{_libdir}
+install -s -m 0755 %{_builddir}/%{githubfull}/lib/libIPSec_MB.so.%{fullversion} %{buildroot}/%{_libdir}
+install -d %{buildroot}/%{_mandir}/man7
+install -m 0444 lib/libipsec-mb.7 %{buildroot}/%{_mandir}/man7
+install -m 0444 lib/libipsec-mb-dev.7 %{buildroot}/%{_mandir}/man7
+cd %{buildroot}/%{_libdir}
+ln -s libIPSec_MB.so.%{fullversion} libIPSec_MB.so.%{major}
+ln -s libIPSec_MB.so.%{fullversion} libIPSec_MB.so
+
+%files
+
+%license LICENSE
+%doc README ReleaseNotes.txt
+
+%{_libdir}/libIPSec_MB.so.%{fullversion}
+%{_libdir}/libIPSec_MB.so.%{major}
+
+%{_mandir}/man7/libipsec-mb.7.gz
```

Nit: This is more easily parsed by the eye without the newlines.

```suggestion
%files
%license LICENSE
%doc README ReleaseNotes.txt
%{_libdir}/libIPSec_MB.so.%{fullversion}
%{_libdir}/libIPSec_MB.so.%{major}
%{_mandir}/man7/libipsec-mb.7.gz
```